app/src/shared/components/Notifications/index.jsx

```diff
@@ -0,0 +1,126 @@
+// @flow
+
+import React, { type Node } from 'react'
+import NotificationSystem from 'react-notification-system'
+
+import BasicNotification from './BasicNotification'
+import TransactionNotification from './TransactionNotification'
+import ModalContext from '$shared/contexts/Modal'
+import Notification from '$shared/utils/Notification'
+import styles from './notifications'
```

Comment:
How about renaming this file to `notificationStyles`? It's not a `.pcss` file so no need to keep up the convention :)